pkg/rest/blockchainhandler/model.go

```diff
@@ -50,7 +50,10 @@ const (
 // TimeResponse contains the response from the /time request
 type TimeResponse struct {
 	Time string `json:"time"`
+	// Hash is the base64 URL-encoded hash of the block.
 	Hash string `json:"hash"`
+	// PreviousHash is the base64 URL-encoded hash of the previous block's header.
```

Comment:
It would be handy to have a short comment on where PreviousHash gets use (thinking about how to describe in the swagger docs).

Reply:
And would it be a generic field that should be in the upstream Sidetree REST API?

Reply:
> And would it be a generic field that should be in the upstream Sidetree REST API?

Yes, I think this should be standard. Bitcoin also has this field in the header: "hashPrevBlock"